src/vods/CustomVod.js

```diff
@@ -42,7 +42,7 @@ export default function Vod(props) {
         .then((response) => response.json())
         .then((response) => {
           setVod(response);
-          document.title = `${response.id} - xQc`;
+          document.title = `${response.date} - xQc`;
```

Comment:
```suggestion
          document.title = `${response.id} - ${response.date} - xQc`;
```
Might be better to include both, sometimes VODs have the same date